aten/src/ATen/native/native_functions.yaml

```diff
@@ -512,7 +556,7 @@
   dispatch:
     CUDA: _s_copy_from_cuda
 
-- func: _copy_same_type_(Tensor self, Tensor src) -> void
+- func: _copy_same_type_(Tensor(a!) self, Tensor src) -> void
```

Comment:
So, is it true that alias annotation serves two purposes: (1) indicate aliasing has occurred and (2) indicate that the argument in question will mutate?

Reply:
Yes and a lot more when complete.

The full description is [here](https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/operator.cpp#L101).

Reply:
OK, looking forward to when you add this to the README :)